src/Illuminate/Support/Number.php

```diff
@@ -367,6 +367,30 @@ public static function defaultCurrency()
         return static::$currency;
     }
 
+    /**
+     * Generate a random number of the given length.
+     *
+     * @param  int  $length
+     * @return int
+     */
+    public static function random(int $length = 6): int
+    {
+        $maxLength = strlen((string) PHP_INT_MAX);
+
+        if ($length < 1 || $length > $maxLength) {
+            return 0;
+        }
+
+        $min = 10 ** ($length - 1);
+        $max = (10 ** $length) - 1;
+
+        if ($length == $maxLength) {
+            $max = PHP_INT_MAX;
+        }
```

Comment:
This way, you don't have to calculate `$max` twice:
```suggestion
        $max = $length == $maxLength ? PHP_INT_MAX : (10 ** $length) - 1;
```